plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -295,6 +306,8 @@ export class PersonState {
             }
         } catch (e) {
             console.error('handleIdentifyOrAlias failed', e, this.event)
+
+            throw e
```

Comment:
iiuc: This would mean that if someone sent us an identify and we decided not to merge we would drop that event, we shouldn't

Reply:
Why not? Nothing was merged.

Reply:
We should add a test for this: make sure the events gets ingested when merge was blocked (easiest to simulate with illegal ids) and also make sure properties get updated for the distinct_id user.

This is the current behaviour and I don't see why we should change it. We have quite a lot of blocked merges, it's easier to debug if we end up processing that event, also if you think of CDP all events (whenever possible) should be exported.

Reply:
I'm working on adding a test for the case you mentioned.

Reply:
I've dropped this `throw e`, and added a couple of test cases to verify events are being ingested on a failed `$merge_dangerously` and that properties are updated. Let me know if these look like you expected or I should do any updates.

Retrying PluginServer CI a few times due to flaky tests...